operator/pkg/model/translation/ingress/dedicated_ingress.go

```diff
@@ -62,10 +62,35 @@ func (d *DedicatedIngressTranslator) Translate(m *model.Model) (*ciliumv2.Cilium
 			UID:        types.UID(m.HTTP[0].Sources[0].UID),
 		},
 	}
-	return cec, getService(m.HTTP[0].Sources[0]), getEndpoints(m.HTTP[0].Sources[0]), err
+	return cec, getService(m.HTTP[0].Sources[0], m.HTTP[0].Service), getEndpoints(m.HTTP[0].Sources[0]), err
 }
 
-func getService(resource model.FullyQualifiedResource) *corev1.Service {
+func getService(resource model.FullyQualifiedResource, service *model.Service) *corev1.Service {
+	serviceType := corev1.ServiceTypeLoadBalancer
+	ports := []corev1.ServicePort{
+		{
+			Name:     "http",
+			Protocol: "TCP",
+			Port:     80,
+		},
+		{
+			Name:     "https",
+			Protocol: "TCP",
+			Port:     443,
+		},
+	}
+
+	if service != nil && service.Type == string(corev1.ServiceTypeNodePort) {
```

Comment:
I think some checks are missing, what is happening if I set 'blabla'?

Reply:
I agree that logging an error, incrementing a metric, or some other feedback to the user would be useful here.

Reply:
Thanks :100: